crypto/evp/m_sigver.c

```diff
@@ -23,6 +23,7 @@ static int update(EVP_MD_CTX *ctx, const void *data, size_t datalen)
     ERR_raise(ERR_LIB_EVP, EVP_R_ONLY_ONESHOT_SUPPORTED);
     return 0;
 }
+#endif /* !defined(FIPS_MODULE) */
```

Comment:
Why the changes in this file?  These would apply to the FIPS provider's internal libctx and it doesn't need access to these when performing other operations (which is the only reason it includes the EVP bindings).